### ISSUE TITLE
Zabbix Proxy 4.0 wrong value on simple check (fping) - no_agent_availability

### ISSUE THREAD

**Issue by nanosistemi** [opnsense/core#4488]
Using Zabbix Server 4.0 and Zabbix Proxy on OpnSense>=20.7, all simple checks are reported as "PINGABLE" also if they are NOT.
Simple checks are simple ping to check an host reachability, usually performed with template like "no_agent_availability".

Zabbix proxy 4.0 uses fping for "simple check" items.
Starting from 20.7 version (freebsd12) the official fping version on amd64 is fping 5.x.

The output of fping 5.x differs from the fping 4.x (reported in the bottom).

Probably the implementation of the proxy (at least on 4.0) doesn't handle this different output.

A temporary workaround found is to revert on fping 4.2 (not found on opnsense amd64 repositories, nor freebsd12 official ones ):

 `pkg add -f http://builder.newimage.nl/packages/FreeBSD_12-1_amd64-2020Q1/All/fping-4.2.txz`

Bye
---
Davide De Bona
www.nanosistemi.com

FPING5:
`root@NaNo-opn-apu3:# /usr/local/sbin/fping -c1 -t50 -i0 192.168.172.10`
`    192.168.172.10 : [0], timed out (NaN avg, 100% loss)`

`    192.168.172.10 : xmt/rcv/%loss = 1/0/100% `

FPING4:
`root@NaNo-opn-apu3:/tmp/fping-4.2 # /usr/local/sbin/fping -c1 -t50 -i0 192.168.172.10`

`    192.168.172.10 : xmt/rcv/%loss = 1/0/100% `

**Comment by fichtner** [comment]
I heard that was fixed in next zabbix release shipping with 20.7.6. We don’t have an ETA yet other than first half of December.

Cheers,
Franco

**Comment by nanosistemi** [comment]
Tnx Franco, happy to hear this.
